### DISCUSSION TITLE
how can i change font size in TextInput?

### DISCUSSION THREAD

**Discussion by zolotaya-ruda** [hecrj/iced#1986] [Q&A] [answered]
i wanna change size text of _placeholder_ and _value_. How can i change it? I guess I can do it using TextInput's style or font method, do I understand correctly? And maybe there are ready-made examples of this?

**Comment by hecrj** [comment, marked as answer]
See https://docs.rs/iced/latest/iced/widget/struct.TextInput.html#method.size.

**Reply by zolotaya-ruda** [reply]
Oh, i thought the size method is used to change the size of the TextInput itself. Сan I then change the size of TextInput, and change the text frame separately, or can I only change the width in TextInput?
